pandas/tests/scalar/period/test_period_asfreq.py

```diff
@@ -21,6 +21,16 @@ def test_asfreq_near_zero(self, freq):
         tup2 = (prev.year, prev.month, prev.day)
         assert tup2 < tup1
 
+    def test_asfreq_near_zero_weekly(self):
```

Comment:
can rename to test_asfreq?

Reply:
Name follows convention set by the test above it (`test_asfreq_near_zero`)

Reply:
I mean the module name itself.